metron-analytics/metron-profiler/README.md

```diff
@@ -328,6 +328,62 @@ Continuing the previous running example, at this point, you have seen how your p
 
 ## Anatomy of a Profile
 
+### Profiler
+
+The Profiler configuration contains only two fields; only one of which is required.
+
+```
+{
+    "profiles": [
+        { "profile": "one", ... },
+        { "profile": "two", ... }
+    ],
+    "timestampField": "timestamp"
+}
+```
+
+| Name                              |               | Description
+|---                                |---            |---
+| [profiles](#profiles)             | Required      | A list of zero or more Profile definitions.
+| [timestampField](#timestampfield) | Optional      | Indicates whether processing time or event time should be used.
```

Comment:
Can we indicate the default here in the description (for quicker reference)?

Reply:
Sure. Update made.